code/obj/machinery/microwave.dm

```diff
@@ -17,7 +23,9 @@
 	var/operating = 0 // Is it on?
 	var/dirty = 0 // Does it need cleaning?
 	var/broken = 0 // How broken is it???
+	var/cook_time = 200 // The time to wait before spawning the item
```

Comment:
200 what

Reply:
code doc format also

Reply:
Sorry not sure what code doc format means here. I don't see anything relevant in the Goonstation Code Guide?

Reply:
https://spacestation13.github.io/DMByExample/meta/dmdoc.html

Reply:
Ok, I think this is done correctly. Have tried replacing appropriate parts.